crypto/s2n_rsa_pss.c

```diff
@@ -33,9 +33,9 @@
 #include "utils/s2n_safety.h"
 #include "utils/s2n_blob.h"
 
-#if RSA_PSS_SUPPORTED
+#if PSS_CERTS_SUPPORTED
```

Comment:
Can we call this "RSA_PSS_CERTS_SUPPORTED" to keep the RSA context in there? Same for the is_supported function.